rules_python_zip/rules_python_zip.bzl

```diff
@@ -348,9 +348,9 @@ _pyz_script_test = rule(
 )
 
 
-def pyz_test(name, srcs=[], data=[], deps=[], pythonroot=None, force_all_unzip=False,
-    interpreter_path=None,
-    flaky=None, licenses=[], local=None, timeout=None, shard_count=None, size=None):
+def pyz_test(name, srcs=[], data=[], deps=[], pythonroot=None,
+    force_all_unzip=False, interpreter_path=None, flaky=None, licenses=[],
+    local=None, timeout=None, shard_count=None, size=None, tags=[]):
```

Comment:
Oh awesome; can you add a `tags` field to one of the `pyz_test` targets in the `tests` directory so we don't accidentally lose this in the future? I think there is one there that specifies a bunch of the other things like shard_count and size and timeout; if it isn't there it should be. Thanks!